Sandbox/BubbleSortLayer.cpp

```diff
@@ -0,0 +1,78 @@
+#include "BubbleSortLayer.h"
+
+#include <chrono>
+#include <thread>
+
+BubbleSortLayer::BubbleSortLayer()
+        : Aquarius::Layer("Bubble Sort") {}
+
+void BubbleSortLayer::onCreation()
+{
+    Aquarius::Application* app = Aquarius::Application::get();
+    Aquarius::Window* window = app->getWindow();
+
+    int height = window->getHeight();
+    int width = window->getWidth();
+
+    m_Camera = std::make_shared<Aquarius::OrthographicCamera>(1, 0.01, height, width);
+
+    Aquarius::Renderer::Init();
+
+    // Populate the bar heights with random values
+    for (int i = 0; i < 100; i++)
+    {
+        barHeights[i] = rand() % 750 + 30;
```

Comment:
So this generates random numbers between 30 and 750 right?

Reply:
Yes sir, I did 30 so the bar would be easily visible on the screen.

Reply:
Perfect